### DISCUSSION TITLE
Is it ok to use the Ben version of the watch app (V1.2) instead of the latest?

### DISCUSSION THREAD

**Discussion by 44616E** [OpenSeizureDetector/Android_Pebble_SD#116] [02_Q&A] [answered]
We are using this on my 5 year old, so the Ben version sounds ideal! i.e. harder to close and no vibration alerts.
But it is a few versions old, so wondering if it will still be compatible with the main app and if there are any issues that were fixed since that may be an issue.

I couldn't find a change log from Version 1.2 to 1.3.2

**Comment by jones139** [comment, marked as answer]
Yes,  that is the version I am using in our production system.   I can't
remember the changes,  but they are about user interface things so not
relevant to people interested in the 'Ben' version.

**Comment by 44616E** [comment]
Awesome, thanks will try it tonight.
